specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/integrationRuntime.json

```diff
@@ -1323,6 +1323,55 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/integrationRuntimes/{integrationRuntimeName}/stop/8A2431BC39114B709A447C12F99E3BB4": {
```

Comment:
- Is there any other operation allowed on integration runtime? How do users start the integration runtime? Is there any get on start operation as well?